Document/0x05f-Testing-Authentication.md

```diff
@@ -155,10 +155,53 @@ Many Apps do not automatically logout a user, because of customer convenience. T
 
 #### Overview
 
+Password strength is a key concern when using passwords for authentication. Password policy defines controls that end users should adhere to. Password Length, Password Complexity and Password Topologies should properly be included in the Password Policy. A "strong" password policy makes it difficult or even improbable for one to guess the password through either manual or automated means. 
```

Comment:
> defines controls
I am not sure "controls" is the right word here. Maybe "requirements"?

> Password Length, Password Complexity and Password Topologies should properly be included in the Password Policy.
I don't think all these words should start with capitals.

> improbable
Maybe "infeasible" is a better word?